packages/components/src/TooltipTrigger/TooltipTrigger.component.js

```diff
@@ -25,8 +26,6 @@ class TooltipTrigger extends React.Component {
 
 	constructor(props) {
 		super(props);
-		this.onMouseOver = this.onMouseOver.bind(this);
-		this.onFocus = this.onFocus.bind(this);
 
 		this.state = {
 			hovered: false,
```

Comment:
with babel class properties, u can declare state out the constructor it will transpile to this.state